x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -417,6 +418,15 @@ export const RulesList = ({
 
   const handleClearRuleParamFilter = () => updateFilters({ filter: 'ruleParams', value: {} });
 
+  usePageReady({
```

Comment:
I guess it's tracking the performance, does it differentiate between o11y and stack rules page?